src/core/multi/MultipleBarcodeReader.ts

```diff
@@ -36,8 +36,9 @@ export default /*public*/ interface MultipleBarcodeReader {
 
   /**
    * @throws NotFoundException
+   * @override decodeMultiple
```

Comment:
override
decodeMultiple !== decodeMultipleWithoutHints
will this cause any problems or confusion?